gtsfm/utils/io.py

```diff
@@ -116,6 +121,43 @@ def read_bal(file_path: str) -> GtsfmData:
     return gtsfm_data
 
 
+def read_cameras_txt(fpath: str) -> Optional[List[Cal3Bundler]]:
+    """Read camera calibrations from a COLMAP-formatted cameras.txt file.
+
+    Reference: https://colmap.github.io/format.html#cameras-txt
+
+    Args:
+        fpaths: path to cameras.txt file
+
+    Returns:
+        calibration object for each camera, or None if requested file is non-existent
+    """
+    if not Path(fpath).exists():
+        logger.info("%s does not exist", fpath)
```

Comment:
info or error?

Reply:
Should be fine, since we can fall back to just using the images in the dir, and using EXIF instead. So i think info is fine, not error

Reply:
Cool. Thanks!